crates/bitwarden-exporters/src/cxf/card.rs

```diff
@@ -0,0 +1,136 @@
+//! Credit card credential conversion
+//!
+//! Handles conversion between internal [Card] and credential exchange [CreditCardCredential].
+
+use bitwarden_vault::CardBrand;
+use credential_exchange_types::format::{Credential, CreditCardCredential};
+
+use crate::Card;
+
+impl From<Card> for Vec<Credential> {
+    fn from(value: Card) -> Self {
+        vec![Credential::CreditCard(Box::new(CreditCardCredential {
+            number: value.number.unwrap_or_default(),
+            full_name: value.cardholder_name.unwrap_or_default(),
+            card_type: value.brand,
+            verification_number: value.code,
+            expiry_date: match (value.exp_year, value.exp_month) {
+                (Some(year), Some(month)) => Some(format!("{}-{}", year, month)),
+                _ => None,
+            },
+            valid_from: None,
+        }))]
+    }
+}
+
+impl From<&CreditCardCredential> for Card {
+    fn from(value: &CreditCardCredential) -> Self {
+        let (year, month) = value.expiry_date.as_ref().map_or((None, None), |date| {
+            let parts: Vec<&str> = date.split('-').collect();
```

Comment:
Would str::split_once be simpler here? https://doc.rust-lang.org/std/primitive.str.html#method.split_once

Reply:
Not really,

```rust
              value
                .expiry_date
                .as_ref()
                .map_or((None, None), |date| match date.split_once('-') {
                    Some((year, month)) => (
                        Some(year).filter(|s| !s.is_empty()),
                        Some(month).filter(|s| !s.is_empty()),
                    ),
                    None => (None, None),
                });```